common/src/main/java/org/conscrypt/SSLParametersImpl.java

```diff
@@ -383,6 +390,19 @@ void setOCSPResponse(byte[] response) {
         return newProtocols.toArray(EMPTY_STRING_ARRAY);
     }
 
+    private static String[] filterFromCipherSuites(String[] cipherSuites, Set<String> toRemove) {
+        if (cipherSuites == null) {
+            return cipherSuites;
+        }
+        ArrayList<String> newCipherSuites = new ArrayList<String>();
```

Comment:
size the ArrayList by cipherSuites.length

Reply:
Done.